extensions/interactions/Continue/directives/continue-interaction.directive.html

```diff
@@ -1,4 +1,11 @@
 <md-button ng-if="$ctrl.isInEditorMode" class="oppia-learner-confirm-button">
   <[$ctrl.buttonText]>
-  <i class="fas fa-arrow-right" style="font-size: 19px; padding-top: 1.5px;"></i>
+  <i class="fas fa-arrow-right oppia-arrow-right-icon"></i>
 </md-button>
+
+<style>
+  continue-interaction .oppia-arrow-right-icon {
```

Comment:
Not sure but I don't think this would be the tag name of the interaction. Make sure to test these changes!

Reply:
This was working fine with this tag name but changed the name now.

Reply:
How it was working fine with the tag name? Do we have any such tag? (Just wanted to check we aren't missing anything!)

Reply:
I didn't find the tag with this name. But surprisingly there was no sideeffect on the styling.

Reply:
That really is odd, and deserves a bit more investigation. 

- If you change the tag to cihuisndsa-interaction (or something random) -- does it work?
- If you change the style to something really obvious/glaring -- does it show up?

Reply:
The mistake is on my side, There is a very minor change in arrow position that I missed to catch the difference. I found it after I changed the background color of arrow to black and found that the position and font size is different, but the difference was minor. Sorry about it.
![Screenshot from 2020-08-19 20-32-02](https://user-images.githubusercontent.com/29057268/90653189-82599900-e25c-11ea-8bbc-7ac2657aa7a1.png)

Reply:
Great -- thanks for checking!